shared/src/commonMain/kotlin/io/customer/shared/util/DateTimeUtil.kt

```diff
@@ -5,22 +5,22 @@ import kotlinx.datetime.Instant
 
 internal interface DateTimeUtil {
     val now: Instant
-    val nowUnixTimestamp: Long
+    val nowEpochSeconds: Long
+    val nowEpochMilliseconds: Long
```

Comment:
```suggestion
    val nowSeconds: Long
    val nowMilliseconds: Long
```

Do we need `epoch` in the naming here?

Reply:
Also, I recommend that we do not introduce new properties to this interface. 

I say this because `DateTimeUtil` was created for automated tests. The test function can mock the SDK code to define the time right now:

```kt
class TestClass {
  fun testFunction() {
    dateUtilStub.now = Date("2022-12-12T11:11:00")
    ...
  }
}
```

`DateTimeUtil` is the interface that is mocked in the test function. 

If we introduce more properties to the interface that needs mocked, it makes the test functions harder to write. 

We would need to modify our test functions to mock multiple properties:
```kt
class TestClass {
  fun testFunction() {
    dateUtilMock.now = Date("2022-12-12T11:11:00")
    dateUtilMock.nowSeconds = ...
    dateUtilMock.nowMilliseconds = ...
    ...
  }
}
```

I do understand that just having just `DateTimeUtil.now` isn't very convenient and it would be nice if we could easily get seconds or milliseconds in our code. 

I think we can introduce that convenience using extensions? 

```kt
// Instead of our SDK code using
dateTimeUtil.nowMilliseconds

// It still uses just .now, but we can easily get milliseconds out of it
dateTimeUtil.now.milliseconds
```

I know KMM code doesn't have automated tests yet. Once it does, I think changes like this will be easier to catch.